packages/sql/src/index.ts

```diff
@@ -35,14 +35,33 @@ const SqlPlugin: Plugin<AST | string> = {
   },
   printers: {
     sql: {
-      print(path, { type, database, finalNewline, ...options }: SqlOptions) {
+      print(
+        path,
+        { type, database, finalNewline, endOfLine, ...options }: SqlOptions,
+      ) {
         const value = path.getValue()
-        const formatted =
+        let formatted =
           typeof value === 'string'
             ? format(value, options)
             : parser.sqlify(value, { type, database })
-        return finalNewline && !formatted.endsWith('\n')
-          ? formatted + '\n'
+
+        const ending = (() => {
```

Comment:
I don’t think a immediate-called function is required here, in `case` just change `ending` instead, or a constant mapper.